wasm2c/wasm-rt-impl.c

```diff
@@ -42,7 +42,7 @@
 #endif
 
 #define PAGE_SIZE 65536
-#define MAX_EXCEPTION_SIZE PAGE_SIZE
+#define MAX_EXCEPTION_SIZE 4096
```

Comment:
This is the used to store the value types that make up the exception.   In all the cases I know of so far this is just a single `i32`.   Larger exception types can be imagined but 4096 seems like rather a excessive.   How about just 256 for now (i.e. 64 i32s or 32 i64s)?   We have a trap in place if this ever get exceeded.

Reply:
256 is fine with me.

Reply:
fixed